docs/configuring-metabase/caching.md

```diff
@@ -54,9 +56,9 @@ On [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.meta
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 51]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Repeated sentence here<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/metabase/metabase/45584?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->